docs/UserGuide.md

```diff
@@ -127,6 +127,7 @@ Examples:
 #### Tips
 
 * A customer can have any number of tags (including 0).
+* In the rare situation where there are two customers with the same name, you can distinguish them by adding an index number behind their names. For example, if there are two `John Smiths`, you could enter them as `John Smith 1` and `John Smith 2`.
```

Comment:
Is 'John Smiths' correct ? so 1 John Smith 2 John Smiths? 1 Sutong 2 Sutongs?

Reply:
maybe if u want do that put the s out side like `John Smith`s

Reply:
Ok